grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -155,6 +155,15 @@ static void startEmbeddedCassandra() {
         }
     }
 
+    static String randomKeyspace(){
+        if (usingOrientDB()) {
+            return "memory";
```

Comment:
The URI has to be "memory" for the moment but that is only for unit tests in the factory module.